src/components/AvatarButton.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { oneOfType, oneOf, string, number } from 'prop-types'
+import glamorous from 'glamorous'
+
+import { colors, transition, radii } from '../theme'
+import { toAlpha } from '../utils/style'
+import Avatar from './Avatar'
+
+const UnstyledButton = glamorous.button(props => ({
```

Comment:
Just a thought, would it make sense to add this as an option for the `<Button>` component.

Reply:
What if we split this out into its own component called `Clickable`.